src/test/java/org/folio/marc4ld/mapper/Bibframe2MarcMapperIT.java

```diff
@@ -31,4 +37,29 @@ void map_shouldReturnCorrectlyMappedMarcJson() {
     assertThat(result).isEqualTo(expectedMarc);
   }
 
+  @ParameterizedTest
```

Comment:
suggest to move this tests to separate  package
`mapper/field999

Reply:
fixed